tests/Unit/ControlSystem/Test_TimescaleTuner.cpp

```diff
@@ -280,6 +281,29 @@ SPECTRE_TEST_CASE("Unit.ControlSystem.TimescaleTuner.NoChangeToTimescale",
   run_tests(-1.0);  // test negative Q
 }
 
+
+SPECTRE_TEST_CASE("Unit.ControlSystem.TimescaleTuner.EqualityAndSerialization",
+                  "[ControlSystem][Unit]") {
```

Comment:
Convert the (non-error) tests to functions and call them from one test case.  This significantly improves performance on some clusters.